examples/datadog-express-apm/host-agent/src/sdk-v3.js

```diff
@@ -16,24 +18,27 @@ const authMiddlewareOptions = {
     },
   },
   scopes: [`manage_project:${projectKey}`],
-  fetch,
+  httpClient: fetch,
 }
 
 const httpMiddlewareOptions = {
   host: 'https://api.europe-west1.gcp.commercetools.com',
   includeRequestInErrorResponse: false,
   includeOriginalRequest: true,
-  fetch,
+  httpClient: fetch,
 }
 
-// datadog options
-const agent = require('path').join(__dirname, '..', 'tracer.js')
+// newrelic options
 const telemetryOptions = {
   createTelemetryMiddleware,
   userAgent: 'typescript-sdk-middleware-datadog',
-  tracer: require(agent).init(),
-  // apm: () => require('dd-trace').init()
+  tracer: () => tracer.init(),
+  apm: () => require('dd-trace').init(),
```

Comment:
There is a file in this folder called `agent.js` which contains one line: `module.exports = require('dd-trace')`. Is the purpose to use this file only and not require `dd-trace` directly? I'm actually still not sure why we need this file. Can you check?

Reply:
yes, you are right. I believe it was required since there was a default value in the ts-sdk-apm for apm/tracer. I'm gonna remove the agent.js file.